subsys/bluetooth/host/att.c

```diff
@@ -237,6 +240,10 @@ static int chan_send(struct bt_att_chan *chan, struct net_buf *buf)
 			return -EINVAL;
 		}
 
+		#if defined(CONFIG_BT_TESTING)
+		atomic_set(&flag_send_over_eatt, (atomic_t) true);
+		#endif
```

Comment:
This seems to have been misplaced and will always be set to true, even for unenhanced ATT bearers

Reply:
Isn't it outside the `atomic_test_bit(chan->flags, ATT_ENHANCED)` check, and thus always set, even for the fixed ATT channel?

Reply:
My bad; It was a combination of Firefox and Github poorly showing the right indentation level, so it looked like it was set for both enhanced or unenhanced bearers, but with a closer look it does look correct.